venv/CalculationClass.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+
+#Query Analysis Calculation Class
+class Calculations():
+
+    def __init__(self,  dataframe,query):
+        self.dataframe = dataframe
+        self.query = query
+
+    def perform_calculations_yearly(self):
+
+                 if self.query=="high_temp":
```

Comment:
Extra tabs. Formatting issue again.

Reply:
Read [PEP-8](https://www.python.org/dev/peps/pep-0008/) for python style guide. Read it thoroughly. I won't be pointing out mistakes related to PEP-8 from now on.